### ISSUE TITLE
Example app build issues

### ISSUE THREAD

**Issue by neodave** [Yummypets/YPImagePicker#280]
I am getting a "Command PhaseScriptExecution failed with a nonzero exit code" trying to build the example app in xCode 10.1. My coworker is able to build fine, so not sure what is going on. I've tried several suggestions I found online that haven't helped. Anyone else encounter this?

**Comment by s4cha** [comment]
@neodave There is a swiftlint run script phase, maybe you could try installing [SwiftLint](https://github.com/realm/SwiftLint) and report back?
Hope this helps

**Comment by neodave** [comment]
I suspected that might be the case, but did not want to install SwiftLint. However I did install the framework in our app and replaced two other frameworks we were hacking together. Overall this is a very impressive project. Having a bit of confusion on customization though, and not getting results I expect. Will keep digging. 
One thing we look to add (and share back) is a circular overlay on a square crop for our avatar needs. other than that it has been plug and play for the most part.  Thank you so much for sharing the code, and for putting it under the MIT license

**Comment by s4cha** [comment]
@neodave You're welcome, don't hesitate the reach out if needed :)
Have you been able to solve the original issue?

**Comment by neodave** [comment]
Honestly I haven't tried to solve the example app issue since I got the framework working in my app natively. Again, great framework. I've been in charge of picking our solution, and trust me I've poked around almost every repo here for image pickers. This is the best one

**Comment by s4cha** [comment]
@neodave Thank you so much for the kind words :)

**Comment by jasonnmark** [comment]
Still not working... :(

**Comment by jasonnmark** [comment]
I installed SwiftLint with brew... but I don't know what to do with it. Not surprisingly just installing it didn't do anything... :) My steps are:
1) download the git repository
2) brew install swiftlint
3) open YPImagePickerExample.xcodeproj in Xcode
4) set target to iPhone Xr (I also tried others)
5) Click on "build and run" button

RESULT: Build Failed with 11 warnings and 1 error.

6) click on the { ! } 

RESULT: Taken to to General settings where the team was a red error. 

7) switch the team to a valid team (mine)
8) Click on "build and run" button

RESULT: Build Failed with 11 warnings and 1 error.

Here's the error:

```
Command PhaseScriptExecution failed with a nonzero exit code

:-1: /Users/xxxx/Library/Developer/Xcode/DerivedData/YPImagePickerExample-bnazbxdqgjgffvebkloiwjkamouv/Build/Intermediates.noindex/YPImagePickerExample.build/Debug-iphonesimulator/YPImagePickerExample.build/Script-99BEB97920A9BD1300FFEB55.sh: line 2: /Users/xxxx/XCode: No such file or directory
```

**Comment by jasonnmark** [comment]
I tried installing CocoaPods and Carthage but I'm not actually sure what I'm supposed to do with them. Are they necessary?

**Comment by jasonnmark** [comment]
I tried the branch called "swift5" and the build no longer fails (still 11 warnings but no errors), but it didn't open the simulator. I finally found that by changing the Scheme (?) on "swift5" to the "example" project (it defaulted to image picker) and it works. Didn't *seem* to work for the latest build though.
![Screen Shot 2019-05-02 at 9 34 33 PM](https://user-images.githubusercontent.com/2500392/57116316-555d3c00-6d22-11e9-89d7-4d558ca35f0c.png)
